README.md

```diff
@@ -29,7 +29,7 @@ build.cmd
 
 The `install` task
 will publish the package
-to Sublime Text's *Data* directory.
```

Comment:
The idea is to do semantic splitting, not based on lenght, etc. Splitting like this makes lines really short and awkward, but if there aren't meant for reading unformatted, then it's fine.